inc/ui/namespace.php

```diff
@@ -133,6 +133,11 @@ function get_current_set_id() : string {
 	$sets = Documentation\get_documentation_sets();
 	$default = array_keys( $sets )[0] ?? '';
 
+	// return early if page is not set.
+	if ( ! isset( $_GET['page'] ) ) {
```

Comment:
What if $_GET[‘set’] is defined?

Reply:
The only time `$_GET[‘set’]` is defined is along with `$_GET[‘page’]`, I don't believe it can be on it's own. 

The return of the complete function appears to return the `$_GET[‘set’]` value first and then `$_GET[‘page’]` :
https://github.com/humanmade/altis-documentation/blob/062fc16095fbce28b255d612a91361af78f5e317/inc/ui/namespace.php#L132-L148

Reply:
That's why I was wondering, if `$_GET['set']` is defined but `$_GET['page']` is not it'll never return the defined `$_GET['set']` parameter. Are we completely sure they're always both supplied?

Reply:
Looking through the docs I haven't identified a situation where the URL contains the `set` on it's own.

URL's with no `set` value appear as:
`wp-admin/admin.php?page=altis-documentation`

Once you have navigated to a different document the URL appears like so, which include a `set`:
`wp-admin/admin.php?page=altis-documentation&set=dev-docs&group=guides&id`

Reply:
Ok, I think we're alright in this instance then but I would still avoid coding in that way if the function expected to be able to return a specific value without that particular condition being met.

Reply:
Would something like this have been better:

```php
if ( ! isset( $_GET['set'] || ! isset( $_GET['page'] ) ) {
```

Reply:
I think && would be correct there

Reply:
Got a PR setup for it here - https://github.com/humanmade/altis-documentation/pull/425